extensions/machine-learning/src/modelManagement/interfaces.ts

```diff
@@ -18,6 +18,7 @@ export interface ListWorkspaceModelsResult extends Array<WorkspaceModel> {
  */
 export interface WorkspaceModel extends Resource {
 	framework?: string;
```

Comment:
nothing for this PR per se but it seems bit odd that this interface has no required types... is that intensional?